documentation/repo-orga.adoc

```diff
@@ -0,0 +1,32 @@
+
+This section  provides an overview of the repository structure, detailing the contents and purpose of each directory and file.
+
+==== Root 
+
+*README.adoc*: This main documentation file, providing an introduction and overview of the project.
+
+*LICENSE*: The licensing information for the project.
+
+*build.gradle*: The build configuration file for Gradle projects.
+
+==== /build
+Contains the files generated by the (gradle) build process. 
+
+==== /docs
+This folder contains the learning goals and everything else needed to build (_generate_) the final curriculum.
+
+==== /documentation
+
+Contains all kinds of documentation about our requirements, build- and release-processes, team etc.
+
+==== /html-thema
+
+t.b.d
```

Comment:
You can also add a description for the license-copyright submodule. Please re-add it to this repository. I'll then adjust the link. 

GitHub cannot resolve asciidoc-includes, regardless of submodule or in the same repository.

Reply:
GitHub failing to resolve the asciidoc includes:
<img width="876" alt="Screenshot 2024-08-21 at 13 24 01" src="https://github.com/user-attachments/assets/a685e5fa-664c-43d5-a709-1bdb816e3868">

Reply:
I have to use the f*** word here - sorry for that... Will re-organize the readme back to the overly monolith state, as possible workarounds (post-commit-hooks or similar) seem overly complicated.